platforms/ios/www/index.html

```diff
@@ -81,7 +81,7 @@
             url = url.replace("#\/redirect", "");
             var splits = url.split('?');
             url = splits[0] + '#\/redirect' + splits[1];
-            window.location.href = url;
+            url = window.location.href;
```

Comment:
Storing the arbitrary URL parameter which can be a user input caused the javascript execution. For my localhost this fix didn't break anything. I am not sure if removing them will break something. I think I will wait for the dev to respond to this .